### PR TITLE
deps: update to use rollkit v0.6.0

### PR DESCRIPTION
**Pull request by tzdybal**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by jcstein — APPROVED** [review]
Looks good to me. Thank you @tzdybal

File: testutil/network/network.go [hunk 1]
```diff
@@ -636,9 +636,9 @@ func (n *Network) Cleanup() {
 			defer cancelFn()
 
 			if err := v.jsonrpc.Shutdown(shutdownCtx); err != nil {
-				v.tmNode.Logger.Error("HTTP server shutdown produced a warning", "error", err.Error())
+				// v.tmNode.Logger.Error("HTTP server shutdown produced a warning", "error", err.Error())
 			} else {
-				v.tmNode.Logger.Info("HTTP server shut down, waiting 5 sec")
+				// v.tmNode.Logger.Info("HTTP server shut down, waiting 5 sec")
 				select {
```
Comment (nashqueue) [inline testutil/network/network.go:642]: Why are we removing those error messages?

**Review by nashqueue — CHANGES_REQUESTED** [review]

File: testutil/network/network.go (hunk 1 above)
Comment (tzdybal) [reply testutil/network/network.go:642]: Because we don't have logger in Node interface, and this is only a test.

**Review by nashqueue — APPROVED** [review]

**Comment by jcstein** [comment]
Testing rollkit installation was successful on this branch:

- [ ] [installation of ethermintd](https://app.warp.dev/block/Xxr4TU63TpPXOpSMtVFPD6) & [running `ethermintd`](https://app.warp.dev/block/9YkelD1IQ7getL2uYtXU8u)
- [ ] [set namespaceid and block height](https://app.warp.dev/block/pC0jXyZtEOPja5tJXIQx7O)
- [ ] [`bash init.sh`](https://app.warp.dev/block/DzPxt6m53LKdKF5iROTmPU)
- [ ] [running the chain connected to mocha node](https://app.warp.dev/block/3U7NkCBMXE9BrJcZaukjb3) 
- [ ] [wallet balance 100000 utia](https://app.warp.dev/block/oJamurmUzar7JO4pkFCNdk) -> [964000](https://app.warp.dev/block/4a9jwhTJeLzA75wzdwiyRg)

**Pull request merged**